src/main/resources/interface-specification.yml

```
@@ -7711,7 +7785,21 @@ components:
           type: string
           format: uuid
       required:
-        - purposeId    
+        - purposeId
+    ClientSeed:
+      description: Client creation request body
+      type: object
+      properties:
+        name:
+          type: string
+          minLength: 5
+          maxLength: 60
+        description:
+          type: string
+          minLength: 10
+          maxLength: 250
```

Constraints are on the process
```suggestion
        description:
          type: string
```